packages/api/src/routes/medical/patient.ts

```diff
@@ -242,21 +257,30 @@ router.get(
 router.post(
   "/:id/consolidated",
   asyncHandler(async (req: Request, res: Response) => {
+    // limit the payload size that can be created
+    const contentLength = req.headers["content-length"];
+    if (contentLength && parseInt(contentLength) >= MAX_CONTENT_LENGTH_BYTES) {
+      return res.status(status.BAD_REQUEST).json({
+        message: `Cannot create bundle with size greater than ${MAX_CONTENT_LENGTH_BYTES} bytes.`,
+      });
+    }
+
     const cxId = getCxIdOrFail(req);
     const patientId = getFrom("params").orFail("id", req);
     const fhirBundle = bundleSchema.parse(req.body);
     const patient = await getPatientOrFail({ id: patientId, cxId });
     const validatedBundle = validateFhirEntries(fhirBundle);
 
-    if (Config.isSandbox()) {
-      const data = await getConsolidatedPatientData({ cxId, patientId });
-      // limit the amount of resources stored in sandbox mode
-      if (data.entry && data.entry.length >= MAX_RESOURCE_STORED_LIMIT) {
-        return res.status(status.BAD_REQUEST).json({
-          message: `Cannot create bundle with size greater than ${MAX_RESOURCE_STORED_LIMIT} bytes in Sandbox mode.`,
-        });
-      }
-    }
+    // TODO #870 review this
+    // if (Config.isSandbox()) {
+    //   const data = await getConsolidatedPatientData({ patient: { id: patientId, cxId } });
+    //   // limit the amount of resources stored in sandbox mode
+    //   if (data.entry && data.entry.length >= MAX_RESOURCE_STORED_LIMIT) {
+    //     return res.status(status.BAD_REQUEST).json({
+    //       message: `Cannot create bundle with size greater than ${MAX_RESOURCE_STORED_LIMIT} bytes in Sandbox mode.`,
+    //     });
+    //   }
+    // }
```

Comment:
Will review this on a diff PR

Reply:
OOC - why are we commenting this out?

AFAIK we still want this so that users can't flood the FHIR server with massive payloads.

Reply:
We can't call `getConsolidatedPatientData` anymore, it doesn't return the payload.

Reply:
I see - I left my comment before I saw 'Will review this on a diff PR"